src/cli/README.md

```diff
@@ -824,6 +825,12 @@ Partition Id Changes: 1
 Better Partition Attach Attempts: 0
 Parent Changes: 0
 Done
+> counters ip
+IPv6 packets successfully transmitted: 15
+IPv6 packets successfully received: 10
+IPv6 packets failed to transmit: 1
+IPv6 packets failed to receive: 0
```

Comment:
```suggestion
TxSuccess: 15
TxFailed: 1
RxSuccess: 10
RxFailed: 0
```

Follow the same naming style of `counters mac` ?

Reply:
if that is the naming convention, will update. But isn't a verbose output more desirable?

Reply:
> But isn't a verbose output more desirable?

The OpenThread project has a general goal to minimize code size. Reducing string length helps achieve that goal.

Reply:
Agree with @jwhui .

Moreover, a more consistent and concise output makes it easier for scripts to interact with OT CLI.